knowledge_graph_queries/core.py

```diff
@@ -1,7 +1,79 @@
 #!/usr/bin/python
-from .queries import QUERIES
+import stardog
```

Comment:
We need to add the package to the requirements file

Reply:
oki